_episodes/67.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: episode
+title: "67: Extra speed with Exasol and Trino"
+date: 2024-10-21
+tags: trino exasol query performance olap
+youtube_id: "qH61QQJKYPA"
+wistia_id: "update date to 24"
+sections:
+- time: 0:00
+  title: Introduction
+- time: 0:53
+  title: Trino releases 461, 462, and 463
+- time: 9:30
+  title: Guests introductions
+- time: 1:00:42
+  title: Upcoming events and rounding out
+introduction: |
+  Manfred is joined by Thomas Bestfleisch from the Exasol team to talk about Exasol,
```

Comment:
```suggestion
  Manfred and Cole are joined by Thomas Bestfleisch from the Exasol team to talk about Exasol,
```
:)

Reply:
Oopps.. thats from before I knew if you would join ;-)